staging/src/k8s.io/kube-scheduler/config/v1beta1/types.go

```diff
@@ -53,12 +53,6 @@ type KubeSchedulerConfiguration struct {
 	// ClientConnection specifies the kubeconfig file and client connection
 	// settings for the proxy server to use when communicating with the apiserver.
 	ClientConnection componentbaseconfigv1alpha1.ClientConnectionConfiguration `json:"clientConnection"`
-	// HealthzBindAddress is the IP address and port for the health check server to serve on,
```

Comment:
We can't remove them from v1beta1; only from the new, not yet released v1beta2.

Reply:
I was too thorough, reverted.

Reply:
lets update the comment here to say that those fields are deprecated and setting them will fail validation.

Reply:
is the secured address and port not configurable via the scheduler config file?

Reply:
They weren't no. I think if we are to add them, we should use a common type for all components like `LeaderElectionConfiguration` and `ClientConnectionConfiguration`

Reply:
@ahg-g  is v1beta1 being removed in 1.23?

Reply:
We didn't discuss it, we are planning a v1beta3 to simplify the plugin configs and tune the weights, so perhaps it is better to remove it to reduce clutter. But i suggest to not block this pr on that.